src/screens/DomainScreen/elements/RenderItem.js

```diff
@@ -315,19 +337,19 @@ const styles = StyleSheet.create({
     borderBottomWidth: 0.5,
     borderBottomColor: COLORS.gray1,
     paddingBottom: 8,
-    paddingTop: 8
+    paddingTop: 8,
   },
   wrapperTextUnFollow: {
-    backgroundColor: COLORS.holyTosca,
+    backgroundColor: '#00ADB5',
     borderRadius: 8,
-    borderColor: COLORS.holyTosca,
+    borderColor: '#00ADB5',
     width: 36,
     height: 36,
     justifyContent: 'center',
     alignItems: 'center',
-    borderWidth: 0.5
+    borderWidth: 0.5,
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `wrapperTextUnFollow` style has been changed from `COLORS.holyTosca` to `#00ADB5`. Ensure that this color change is intentional and consistent with the rest of your application's color scheme.


<!-- This is an auto-generated comment by OSS CodeRabbit -->